validate-swupd.sh

```diff
@@ -43,11 +43,11 @@ for i in $(seq $1 10 $2); do
 		echo "failed to init mix"
 		exit 1
 	fi
-	mixer bundle add --all-upstream
-	if [[ $? -ne 0 ]]; then
-		echo "failed to add mix bundles"
-		exit 1
-	fi
+	#mixer bundle add --all-upstream
+	#if [[ $? -ne 0 ]]; then
+	#	echo "failed to add mix bundles"
+	#	exit 1
+	#fi
```

Comment:
Any reason to leave this commented out in the code?

Reply:
fixed